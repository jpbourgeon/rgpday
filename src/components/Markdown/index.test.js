/* global describe, it, expect */
import React from 'react'
import renderer from 'react-test-renderer'
import Markdown from './'
import jssSerializer from 'src/utils/jssSnapshotSerializer'

expect.addSnapshotSerializer(jssSerializer)

describe('The Markdown component', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<Markdown>{`# Hello world`}</Markdown>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
