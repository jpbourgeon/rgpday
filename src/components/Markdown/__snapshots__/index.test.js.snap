// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Markdown component should render correctly 1`] = `
<h4
  className="root-1 h4-16 gutterBottom-28"
  id="hello-world"
>
  Hello world
</h4>
`;
